addons/udes_stock/models/stock_picking_type.py

```diff
@@ -12,6 +15,9 @@
 class StockPickingType(models.Model):
     _inherit = "stock.picking.type"
 
+    # Overwrite sequence_code as it is only needed for old odoo prefixes.
```

Comment:
NIT: we should capitalise "Odoo" when it's being used as the name of the framework.